test/lib/ufe/testDuplicateCmd.py

```diff
@@ -183,5 +186,24 @@ def childrenNames(children):
         self.assertNotIn(ballDupName, propsChildrenNames)
         self.assertEqual(ballDupName, "Ball_36")
 
-        cmds.undo() # undo duplication
-        cmds.undo() # undo deletion
+        cmds.undo()  # undo duplication
+        cmds.undo()  # undo deletion
+
+    @unittest.skipUnless(mayaUtils.previewReleaseVersion() >= 122, 'Requires Maya fixes only available in Maya Preview Release 122 or later.')
+    def testSmartTransformDuplicate(self):
+        '''Test smart transform option of duplicate command.'''
+        torusFile = mayaUtils.getTestScene("groupCmd", "torus.usda")
+        torusDagPath, torusStage = mayaUtils.createProxyFromFile(torusFile)
+        usdTorusPathString = torusDagPath + ",/pTorus1"
+
+        cmds.duplicate(usdTorusPathString)
+        cmds.move(10, 0, 0, r=True)
+        smartDup = cmds.duplicate(smartTransform=True)
+
+        usdTorusItem = ufeUtils.createUfeSceneItem(torusDagPath, '/pTorus3')
+        torusT3d = ufe.Transform3d.transform3d(usdTorusItem)
+        transVector = torusT3d.inclusiveMatrix().matrix[-1]
+
+        correctResult = [20, 0, 0, 1]
+
+        self.assertEqual(correctResult, transVector)
```

Comment:
Test is to just duplicate, make a move, and duplicate with the smartTransform flag without changing selection at all.  In this case it should apply the same move to the second new object.

I added a couple simple files for the other test I'm adding and realized later that I'm using those files in this test as well, which wasn't intentional but should be fine, it's just a torus and a sphere.